packages/openapi/README.md

```diff
@@ -0,0 +1,65 @@
+# OpenAPI 3.1 Validator
```

Comment:
`openapi` _could_ be in a standalone repo since it's not Open Payments specific despite using and Open Payments spec for testing

Reply:
I think it's fine in here for now to keep things simple